esp_flash_dispatcher/test_apps/flash_dispatcher/main/test_esp_flash_dispatcher.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Unlicense OR CC0-1.0
+ */
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <freertos/FreeRTOS.h>
+#include <freertos/task.h>
+#include <freertos/semphr.h>
+
+#include <unity.h>
+#include "esp_flash.h"
+#include <esp_attr.h>
+#include "esp_log.h"
+#include "unity.h"
+#include "sdkconfig.h"
+
+// Buffer for PSRAM task stack
+static StackType_t *s_psram_task_stack = NULL;
+
+#define TEST_FLASH_ADDRESS 0x110000 // default address in flash for testing
```

Comment:
It's probably safer to create a partition and get the address of that partition?